docs/design/ovs-pipeline.md

```diff
@@ -1,1197 +1,1852 @@
 
-* Tunnelled traffic coming-in from a peer Node and destined to a local Pod is
-  directly forwarded to the Pod. This requires setting the source MAC to the MAC
-  of the local gateway interface and setting the destination MAC to the Pod's
-  MAC address. Then the packets will go to [L3DecTTLTable] for decrementing
-  the IP TTL value. Such packets can be identified by bit 19 of the NXM_NX_REG0
-  register (which was set to 1 in the [ClassifierTable]) and the destination IP
-  address (which should match the IP address of a local Pod). We therefore
-  install one flow for each Pod created locally on the Node. For example:
+Flows 3-6, whose priorities are all 14500, are installed for the egress rule `AllowToDB` in the sample policy. These
+flows are described as follows:
 
-```text
-table=70, priority=200,ip,reg0=0x80000/0x80000,nw_dst=10.10.0.2 actions=mod_dl_src:e2:e5:a4:9b:1c:b1,mod_dl_dst:12:9e:a6:47:d0:70,goto_table:72
-```
+- Flow 3 is used to match packets with the source IP address in set {10.10.0.24}, which has all IP addresses of the Pods
+  selected by the label `app: web`, constituting the first dimension for `conjunction` with `conj_id` 7.
+- Flow 4 is used to match packets with the destination IP address in set {10.10.0.25}, which has all IP addresses of
+  the Pods selected by the label `app: db`, constituting the second dimension for `conjunction` with `conj_id` 7.
+- Flow 5 is used to match packets with the destination TCP port in set {3306} specified in the rule, constituting the
+  third dimension for `conjunction` with `conj_id` 7.
+- Flow 6 is used to match packets meeting all the three dimensions of `conjunction` with `conj_id` 7 and forward them
+  to table [EgressMetric], persisting `conj_id` to `EgressRuleCTLabel` that will be consumed in table [EgressMetric].
 
-* All tunnelled traffic destined to the local gateway (i.e. for which the
-  destination IP matches the local gateway's IP) is forwarded to the gateway
-  port by rewriting the destination MAC (from the Global Virtual MAC to the
-  local gateway's MAC).
+Flows 7-9, whose priorities are all 14499, are installed for the egress rule with a `Drop` action defined after the rule
+`AllowToDB` in the sample policy, serves as a default rule. Unlike the default of K8s NetworkPolicy, Antrea-native
+NetworkPolicy has no default rule, and all rules should be explicitly defined. Hence, they are evaluated as-is, and
+there is no need for a table [AntreaPolicyEgressDefaultRule]. These flows are described as follows:
 
-```text
-table=70, priority=200,ip,reg0=0x80000/0x80000,nw_dst=10.10.0.1 actions=mod_dl_dst:e2:e5:a4:9b:1c:b1,goto_table:80
-```
+- Flow 7 is used to match packets with the source IP address in set {10.10.0.24}, which is from the Pods selected
+  by the label `app: web`, constituting the first dimension for `conjunction` with `conj_id` 5.
+- Flow 8 is used to match any packets, constituting the second dimension for `conjunction` with `conj_id` 5.
```

Comment:
let's just add one sentence about it at the end of the bullet point:

> This flow, which matches all packets, exists because we need at least 2 dimensions for a conjunctive match.

Reply:
Will add